3scale/3scale-operator
# [WIP] Mgdapi 3473

# Issue 
[MGDAPI-3473](https://issues.redhat.com/secure/RapidBoard.jspa?rapidView=6585&view=detail&selectedIssue=MGDAPI-3473)

# What
Added support for product deletion 

# Verification

1.  Install RHOAM locally from this branch: https://github.com/integr8ly/integreatly-operator/pull/2409 but replace the manifests/3scale/0.9.0 values for container image and image from quay.io/austincunningham/3scale-operator:cluster-scoped to quay.io/pstefans/3scale-operator:v0.45.0
2. Wait for the installation to complete
3. Create product via CR
4. Confirm product got created in 3scale
5. Delete product CR
6. Confirm product has been deleted in 3scale
 
Note: Guide for creating products through CRs https://docs.google.com/document/d/16CTVrFtsDR27_ZS9-7_xS9JKwteSmi7I_IFV7DShuXI/edit
